packages/core/src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -117,8 +118,10 @@ export class HttpResourceFetcher implements ResourceFetcher<Response> {
                 return req.response.finally(() => cancelListener?.dispose())
             },
             {
-                timeout: 3000,
-                interval: 100,
+                // Ensures we don't abort a long request (eg: slow connection).
+                // This timeout is just backup, the given Timeout object should be relied on instead.
+                timeout: oneMinute * 30,
```

Comment:
would this mean that all of Thomas' requests we're getting aborted? but if we waited 1.5 minutes they would eventually finish

Reply:
Yeah this was the weird part. Since my requests were getting aborted. I'm going to triple check once more

Reply:
Found out that there is a difference between getting the initial response back (`fetch(...).response`) and `response.body`. `response.body` is called outside of the HttpResourceFetcher and is not subject to this value.

So only the initial response for just the headers is encapsulated in this, the actual body fetching is outside.